app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -18,7 +18,12 @@ header > h1 .label {
   padding: $panel-body-padding;
 }
 
-.relationships,
+.attribute-filename,
+header h1,
+td:first-child {
+  word-break: break-all;
+}
+
 .attributes {
```

Comment:
Did you intend to remove `.relationships` from this selector?

Reply:
@jcoyne - It was a mistake.  I added it back.